features/keychain/module/keychain.js

```diff
@@ -26,7 +26,7 @@ export class Keychain extends ExodusModule {
   #legacyPrivToPub = null
 
   // TODO: remove default param. Use it temporarily for backward compatibility.
-  constructor({ legacyPrivToPub = {}, logger }) {
+  constructor({ legacyPrivToPub = Object.create(null), logger }) {
```

Comment:
Changes look fine, although not sure I fully get the reasons behind it, at least for the default argument cases. We aren't passing those values anywhere, just reading predefined keys from them